tests/acceptance/features/coreApiSharees/sharees.feature

```diff
@@ -248,8 +248,7 @@ Feature: sharees
 
 
   Scenario Outline: search for sharees when search min length is set to 2
-    Given the administrator has updated system config key "user.search_min_length" with value "2"
-    And user "sharee2" has been created with default attributes and without skeleton files
+    Given user "sharee2" has been created with default attributes and without skeleton files
```

Comment:
What is the default "search min length" on oCIS?

We need to put a comment that this scenario is only expected to work if the system-under-test already supports providing only 2 characters in the search string.

Reply:
Note: We have an env variable to set a search length in frontend 
https://github.com/owncloud/ocis/blob/5ad8e283b669a7270e6925bf653a636610e022ed/.drone.star#L1922
 not sure how to do this through API.

Reply:
I think if a test becomes useless because those things are not applicable / not configurable in ocis, we should remove those scenarios completely